src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Validation/HealthCertificateValidationCoordinator.swift

```diff
@@ -0,0 +1,150 @@
+//
+// 🦠 Corona-Warn-App
+//
+
+import UIKit
+import class CertLogic.ValidationResult
+
+final class HealthCertificateValidationCoordinator {
+	
+	// MARK: - Init
+	
+	init(
+		parentViewController: UIViewController,
```

Comment:
Would it be possible to expose a root viewcontroller and let the calling coordinator decide how to present the root of this coordinator. Just hypothetically the coordinator then could be used in different navigation scenarios.

Reply:
I used the same pattern here that we use for the exposure submission coordinator. I see the theoretical freedom of exposing the root view controller, but in the end the whole flow needs to be either presented modally or pushed. And that exposed root view controller would need to already be a navigation controller, and couldn't be pushed onto another one. So the only viable option is to present it modally, and there I don't see an up- or downside for both approaches. So I'm just gonna leave it like it is for now.